.github/workflows/codeql.yml

```
@@ -13,86 +1,128 @@
-          'your code, for example:'
-        echo '  make bootstrap'
-        echo '  make release'
-        exit 1
+      # Run the analysis
+      - name: Perform CodeQL Analysis
+        uses: github/codeql-action/analyze@v3
+        with:
+          category: "/language:${{ matrix.language }}"
+          output: sarif-results
+          upload: true
+          wait-for-results: true
```

actionlint (1.7.7)

input "wait-for-results" is not defined in action "github/codeql-action/analyze@v3". available inputs are "add-snippets", "category", "check_name", "checkout_path", "cleanup-level", "expect-error", "matrix", "output", "ram", "ref", "sha", "skip-queries", "threads", "token", "upload", "upload-database", "wait-for-processing"

(action)